js/queryentrylist.js

```diff
@@ -0,0 +1,132 @@
+const accountAPIURL = "http://localhost:8002/api/accounts";
+const recordAPIURL = "http://localhost:8001/api/records";
+
```

Comment:
Replace these two with the global variable version

Reply:
Changed